keybase/prompt_test.go

```diff
@@ -96,3 +96,31 @@ func TestPromptError(t *testing.T) {
 	assert.Error(t, err)
 	assert.Nil(t, resp)
 }
+
+func testPausedPromptWithCommand(t *testing.T, promptCommand string, timeout time.Duration) (bool, error) {
+	cfg, _ := testConfig(t)
+	ctx := newContext(&cfg, log)
+	assert.NotNil(t, ctx)
+
+	if timeout > 0 {
+		defaultPromptTimeout := promptTimeout
+		promptTimeout = timeout
+		defer func() { promptTimeout = defaultPromptTimeout }()
```

Comment:
Yeah this is smelly... removing the prompt global timeout vars, and making an arg like a sane person :)